app/controllers/reservations_controller.rb

```diff
@@ -4,14 +4,28 @@ class ReservationsController < ApplicationController
   before_action :set_reservation, only: %i[show edit update destroy]
   before_action :authenticate_user!
 
+  def find_reservations_by_user
+    @reservations = Reservation.where(user_id: 
+      User.find_by(email: params[:email]))
```

Comment:
This line is an ideal candidate for another method :wink:

Reply:
So should make private method, let's say `get_user_by_email` and simply call it here right?

Reply:
Yes, it should increase readability.

Reply:
Okey, did that, thank you for all the comments! :D